src/cogent3/core/alignment.py

```diff
@@ -2013,6 +2013,22 @@ def __init__(self, map, data, length=None):
         if hasattr(data, "name"):
             self.name = data.name
 
+    def annotate_matches_to(
+        self, pattern=None, annot_type=None, name=None, allow_multiple=False
```

Comment:
only `allow_multiple` should be a keyword argument, all others must be required.